### PR TITLE
Vet Clinic Database Milestone: Create animals table

### PR DESCRIPTION
**Pull request by CarlosIgreda**
For this Milestone I:

✅ Used GitFlow
✅ Created tables in SQL
✅ Inserted and queried data in SQL

Screenshots:

✅ Find all animals whose name ends in "mon"

image

✅ List the name of all animals born between 2016 and 2019

image

✅ List the name of all animals that are neutered and have less than 3 escape attempts

image

✅ List the date of birth of all animals named either "Agumon" or "Pikachu"

image

✅ List name and escape attempts of animals that weigh more than 10.5kg

image

✅ Find all animals that are neutered

image

✅ Find all animals not named Gabumon

image

✅ Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,291 +1,106 @@
...
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
   - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
```
Comment (topeogunleye) [inline README.md]: - [x] Please you need to put back the headers you removed back into the project because the optional section is the FAQ's

**Review by topeogunleye — COMMENTED** [review]
Hi @CarlosIgreda, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that the README file is not complete. 

- [x] In the case of README consider not leaving out any of the sections in the [Microverse README](https://github.com/microverseinc/readme-template) template as it's a project requirement.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md (hunk 1 above)
Comment (CarlosIgreda) [reply README.md]: @topeogunleye Done it.

**Review by Ibizugbe — APPROVED** [review]
Hi @CarlosIgreda 👋👋👋, 

approve2


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅✅✅

- [X] `animal` table has been created with the commands added to the `schema` file👍.
- [X] Required data has been added to the animal table with the commands updated in the `data.sql` file👍.
- [X] The required queries have been written and updated in the `queries.sql` file👍.
- [X] Result of each queries have been updated on the PR👍.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by CarlosIgreda** [comment]
@Ibizugbe thank you so much for your feedback!

**Pull request merged**